setup.cfg

```
@@ -36,7 +36,7 @@ python_requires = >=3.8
 packages = find:
 zip_safe = True
 install_requires =
-    appdirs>=1.4.4,<1.5.0
+    platformdirs>=3.0.0
```

"Pin all the things!"

```suggestion
    platformdirs>=3.0.0,<3.1.0
```

---

done